site/en/docs/extensions/mv3/manifest/content_scripts/index.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: 'layouts/doc-post.njk'
+title: 'Manifest - Content Scripts'
+seoTitle: 'Chrome Extensions Manifest: content_scripts'
+date: 2023-08-10
+description: Reference documentation for the content_scripts property of manifest.json.
+---
+
+## Overview {: #overview }
+
+The `"content_scripts"` field is used to statically load a JavaScript or CSS file every time a page is opened that matches a certain [URL pattern][doc-match]. Extensions can also inject content scripts programmatically, see [Injecting Scripts][doc-cs] for details.
+
+## Manifest {: #manifest }
+
+These are the supported fields for `"content_scripts"`. Only the `"matches"` key and either `"js"` or `"css"` are required.
+
+{% Label %}manifest.json{% endLabel %}
+
+```json
+{
+ "name": "My extension",
+ ...
+ "content_scripts": [
+   {
+     "matches": ["https://*.example.com/*"],
+     "css": ["my-styles.css"],
+     "js": ["content-script.js"],
+     "exclude_matches": ["*://*/*foo*"],
+     "include_globs": ["*example.com/???s/*"],
+     "exclude_globs": ["*bar*"],     
+     "all_frames": false,
+     "match_origin_as_fallback": false,
+     "match_about_blank": false,
+     "run_at": "document_idle",
+     "world": "ISOLATED",
+   }
+ ],
+ ...
+}
+```
+
+## Files {: #files }
+
+Each file must contain a relative path to a resource in the extension's root directory. Leading slashes (`/`) are automatically trimmed. The [`"run_at"`](#world-timings) specifies when each file will be injected.  
+
+`"css"` - Array
+: _Optional_. An array of CSS file paths, injected in the order of this array, and before any DOM construction or page rendering occurs.
```

Comment:
>_Optional_.

Is this an existing idiom? The italics are good for making it scannable. I want to try to remember that you did it this way.